webview/src/plots/components/comparisonTable/ComparisonTable.test.tsx

```diff
@@ -496,7 +503,12 @@ describe('ComparisonTable', () => {
 
       const [, firstRow, secondRow] = screen.getAllByRole('rowgroup') // First rowgroup is the thead
 
-      dragAndDrop(secondRow, firstRow, DragEnterDirection.BOTTOM)
+      dragAndDrop(
+        secondRow,
+        firstRow,
+        DragEnterDirection.BOTTOM,
+        EventCurrentTargetDistances
```

Comment:
[C] This is gross but I could not find any other way to successfully spy on a single call of `getEventCurrentTargetDistances`. I had to

1. give the function its own module
3. move the import to inside the test file (instead of the util)
4. mock inside the test file for the usual workaround
5. pass the module down into the test helper. 

I had to do this in 4 places.